view/Requirements.php

```diff
@@ -1328,6 +1345,42 @@ public function themedCSS($name, $module = null, $media = null) {
 			$this->css($theme . $css, $media);
 		} elseif($module) {
 			$this->css($module . $css, $media);
+		} elseif (Director::isDev()) {
```

Comment:
As highlighted in #6373, this breaks behaviour that modules/sites were relying on: by silently ignoring missing files, modules used this to optionally include a file if a theme provided. So this is a semver failure.